panther_gazebo/panther_gazebo/launch/panther.launch.py

```diff
@@ -10,35 +10,29 @@
 def generate_launch_description():
   panther_gazebo = get_package_share_directory('panther_gazebo')
   panther_description = get_package_share_directory('panther_description')
-  world = LaunchConfiguration('world', default=os.path.join(panther_description, 'worlds', 'willowgarage.world'))
-  panther_description_config_path = os.path.join(panther_description, 'models', 'panther', 'config')
+  panther_description_config_path = os.path.join(panther_description, 'config')
 
-  panther_common_props_path = LaunchConfiguration('panther_common_props_path', default=os.path.join(panther_description_config_path, 'panther_common.yaml'))
-  wheel_props_path = LaunchConfiguration('wheel_props_path', default=os.path.join(panther_description_config_path, 'WH01.yaml'))
+  wheel_config_path = LaunchConfiguration('wheel_config_path', default=os.path.join(panther_description_config_path, 'WH04.yaml'))
```

Comment:
```suggestion
  wheel_config_path = LaunchConfiguration('wheel_config_path', default=os.path.join(panther_description_config_path, 'WH01.yaml'))
```
I guess WH01 should be default

Reply:
Yeah. My bad